libs/island-ui/core/src/lib/Accordion/AccordionItem/AccordionItem.tsx

```diff
@@ -145,12 +145,16 @@ export const AccordionItem = forwardRef<HTMLButtonElement, AccordionItemProps>(
                     component="span"
                     height="full"
                     width="full"
-                    display="flex"
+                    display="block"
                     alignItems="center"
```

Comment:
one late comment, what is the reason for the display to change to `block`? `alignItems="center"` is probably useless here if we change the display property to block

Reply:
and this could cause some side-effects
e.g. here I'm toggling between block and flex
![2022-05-16 18 05 31](https://user-images.githubusercontent.com/5655741/168655609-c446e9f7-d595-4546-b631-551f81f531c8.gif)

Reply:
I added it because I needed the parent element (the label in this case) to take full with. So I could space these two elements with justify content.
<img width="771" alt="Screenshot 2022-05-17 at 09 55 15" src="https://user-images.githubusercontent.com/5700902/168785111-9c077e9b-7d95-4748-9719-b67d8c3dd931.png">

I fixed the issue in the judicial-system by adding `flex-grow: 1` to the custom label.